ajax/libs/gijgo/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "gijgo",
+  "version": "1.3.1",
+  "description": "Gijgo is a set of free open source javascript controls distributed under MIT License. All widgets are high performance, built on top of the jQuery JavaScript Library and with built-in support for Bootstrap and jQuery UI. They are designed to saves you time and scales with your development process.",
+  "main": "js/gijgo.js",

```

Comment:
@atatanasov We don't need `main`, `bugs`, `scripts`, `dependencies` and `devDependencies` field. Please remove them. Thanks.